content/articles/managing-docker-containers-using-command-line-interface/index.md

```diff
@@ -0,0 +1,97 @@
+
+```bash
+sudo docker run -it ubuntu
+```
+
+### Remove all the dangling volumes
+
+Docker Volumes may be mounted with Docker Containers to share files and directories among many Docker Containers. When you delete Docker Containers, the Docker Volumes linked with them remain. Volumes in Docker are precisely what they sound like. With the following command, you can get a list of all dangling docker volumes:
+```bash
+sudo docker volume ls -f dangling=true
+```
+
+> To avoid leaving dangling volumes while deleting docker volumes, use the `-v flag`.
+
+```bash
+sudo docker rm -v <name-of-the-container>
+```
+
+To list the docker containers that are now executing, use the following sequence of instructions. Stop the specific container and remove it with the -v flag to prevent leaving dangling volumes behind.
+
+>One by one, run the commands below.
+
+```bash
+sudo docker container ls
+sudo docker stop my-container-01
+sudo docker rm -v my-container-01
+```
+
+### Removing docker containers and images
+
+Docker containers are not automatically deleted when they are stopped unless they are started using the `—rm` parameter. However, before deleting a docker container, you must first ensure that it is not operating. Containers can be stopped and deleted with the following instructions:
+
+```bash
+sudo docker ps -a
+sudo docker stop <name-of-the-container>
+sudo docker rm <name-of-the-container> 
+```
+
+The first command lists all of your system's containers. The container's state may be found in the status column. Stopping the container before removing it is required if it has not been exited. You must know the container ID of the container you want to delete before you delete them.
+
+Remove all the containers linked with a docker image before removing it using the command below:
+
+```bash
+sudo docker rmi <The-ID-of-the-image>
+```
+
+### Using aliases
+
+There are times when we must repeat the same command-line instruction, which is made more difficult if the command in question contains parameters, and much more so if we must change the output to make it legible. Most of us end up making some sort of cheat sheet to keep track of all those complicated commands in case we need to execute them again.
+
+We may save time and effort by utilizing an `alias command` to avoid typing or duplicating the same command over and over again. Aliases allow us to execute a command or a group of instructions using a pre-defined `string` that we can tailor to our liking.
+
+To handle aliases, simply create and use aliases in your `/.bashrc file` and mention them.
+
+```bash
+alias dockrm='docker rm'
+alias docklist='docker ps -a'
+```
+
+### Examining Docker containers 
+
+The Docker inspect command may be used to acquire information about a specific docker container. It provides you with all of the information about the container, including the path, creation date, status, driver, and so on. The container name is required to check the container.
+

```

Comment:
Please show us how the docker `inspect` command is used. Don't just state it. This is beneficial to the reader. Also, explain the commands in your article where necessary.

Reply:
The Docker inspect command.....sentence is here. Was it a typo?